docs/en_US/Builtin_Tuner.md

```diff
@@ -323,3 +324,61 @@ tuner:
   classArgs:
     optimize_mode: maximize
 ```
+
+<br>
+
+<a name="BOHBTuner"></a>
+
+![](https://placehold.it/15/1589F0/000000?text=+) `BOHB Advisor`
+
+> Builtin Tuner Name: **BOHB**
+
+**Installation**
+
+BOHB advisor requires [ConfigSpace](https://github.com/automl/ConfigSpace), so users should install it first. User could use `pip3 install ConfigSpace` to install it.
+
+To use BOHB Advisor, you **must report the loss** as one of the final result to update Bayesian Optimization. So you should return a **dict** type variable with metrics in key "default" and loss in key "loss". For example:
```

Comment:
must report the loss?  how about other metrics?

Reply:
The author of the paper uses the loss for BO, and the performance is good. So the use of loss is specified here. I think loss is a common metric to get?

Reply:
Support all metrics now.